docs/portal/client-side-integration.md

```diff
@@ -7,21 +7,25 @@ sidebar_position: 09
 
 # Client-Side Integration
 
-In the UID2 Portal, if you want to use an implementation option that generates UID2 tokens on the client side, you'll need to define one or more of each of the values shown in the following table.
+In the UID2 Portal, if you want to use an implementation option that generates UID2 tokens on the client side, you'll need to define the following values that are applicable to your implementation type:
 
-| Value | Details | Documentation Link |
-| :--- | :--- | :---|
-| Key pair | At least one. In your implementation, you'll share the public key. | [Subscription ID and Public Key](getting-started/gs-credentials.md#subscription-id-and-public-key) |
-| Domain | At least one. Provide a complete list of your root-level domains. | [Client-Side Implementation for Publishers](../getting-started/gs-account-setup.md#client-side-implementation-for-publishers) |
+| Value | Implementation Type | Details | Documentation Link |
+| :--- | :--- | :---| :---|
+| Key pair | All (web or mobile) | At least one. In your implementation, you'll share the public key. | [Subscription ID and Public Key](../getting-started/gs-credentials.md#subscription-id-and-public-key) |
+| Domain |  Web | At least one. Provide a complete list of your root-level domains. | [Client-Side Implementation for Publishers](../getting-started/gs-account-setup.md#client-side-implementation-for-publishers) |
+| Mobile App ID |  Mobile | Any IDs that apply to your mobile apps: Android App ID, iOS/tvOS Bundle ID, or iOS App Store ID. | [Client-Side Mobile Integrations](../getting-started/gs-account-setup.md#client-side-mobile-integrations) |
 
 :::important
 It's important to provide a complete list of root-level domain names for your sites. This is a security measure, for client-side implementation only. If a domain is not defined in the UID2 Portal, UID2 token requests from that domain will fail.
 :::
```

Comment:
I would add a similar note on the importance of adding mobile app ids too for the same reasons

Reply:
Very good point! Done. I expanded this one to include both, and added a line in the "add" section for domains and app IDs.